index.js

```diff
@@ -1,33 +1,44 @@
 
-
-//Getting the different placeholders
-
 let content = document.getElementById("content");
 let integerValue = parseInt(content.innerHTML);
 
-const decreaseButton = document.getElementById("btn-decrease");
-const resetButton = document.getElementById("btn-reset");
-const increaseButton = document.getElementById("btn-increase");
+const arrayButton = document.getElementsByClassName("button-event");
+const buttons = Object.entries(arrayButton);
+buttons.forEach((button, index) => {
+    function changeValue(button, index) {
+        switch (index) {
+            case 0:
+                button = document.getElementById("btn-decrease").addEventListener("click", () => {
```

Comment:
The addEventListener should be added once inside the for loop 

For example: buttons.forEach((button,index) => { button.addEventListener.....})

That means the for loop will go through each button.

Inside the addEventListener you can then call the function by passing the index as argument